daemon/web/src/lib/components/SystemStatsTable.svelte

```diff
@@ -1,12 +1,71 @@
 <script lang="ts">
-    import { type SystemStats } from '$lib/systemStats';
+    import { BatteryLevel, type SystemStats } from '$lib/systemStats';
     let {
         stats,
     }: {
         stats: SystemStats;
     } = $props();
 
     const table_cell_classes = 'border p-1 lg:p-2';
+
+    let battery_level = $derived.by(() => {
```

Comment:
since we apparently need a percentage in the UI i do think it would make more sense to make everything percentage based. even if the orbic module would end up reporting only 5 possible numbers

Reply:
OK, I can work on that. Out of curiosity, what's driving the requirement for percentage point precision?

Reply:
> what's driving the requirement for percentage point precision?

there is no requirement from my side, i just see that the battery icon is ultimately driven by a percentage and not any of the "level states", and that we don't show anything else. so why not pass that percentage from the backend and use it unaltered in the frontend?